.github/workflows/ci.yml

```diff
@@ -85,14 +85,22 @@ jobs:
       max-parallel: 10
       matrix:
         include:
-          - {"salt-version": "3005.0", "python-version": "3.7", "source": "pypi"}
-          - {"salt-version": "3005.0", "python-version": "3.8", "source": "pypi"}
-          - {"salt-version": "3005.0", "python-version": "3.9", "source": "pypi"}
-          - {"salt-version": "3006.0", "python-version": "3.7", "source": "pypi"}
-          - {"salt-version": "3006.0", "python-version": "3.8", "source": "pypi"}
-          - {"salt-version": "3006.0", "python-version": "3.9", "source": "pypi"}
-          - {"salt-version": "3006.0", "python-version": "3.10", "source": "pypi"}
-          - {"salt-version": "3006.x", "python-version": "3.10", "source": "onedir"}
+          - {"salt-version": "3005.0", "python-version": "3.7", "examples": false, "source": "pypi"}
```

Comment:
`true` or `false` is not going to look that good when GH Actions shows it.

Maybe use `testsuite` with values `core` and `examples`?